test/resources/v5/order-recipients.test.js

```diff
@@ -58,32 +63,84 @@ describe('v5 order-recipients', () => {
   });
 
   describe('when configured', () => {
-    describe('#storeOrderRecipient', () => {
-      it('should contain the piiServerUrl in the request URL', async () => {
-        const member = { id: faker.random.number(), accept_language: 'en-CA' };
+    const member = { id: faker.random.number(), accept_language: 'en-CA' };
+    const mockOrderRecipient = {
+      mockOrderRecipientCode,
+      validation_signature: '0t0ir1mXeV4Ii5-RgSZY....xFA==',
```

Comment:
Optional: `faker.random.uuid()`?
```suggestion
      validation_signature: `${faker.random.uuid()}FA==`,
```

Reply:
done